frontend/src/js/api/types.ts

```diff
@@ -428,9 +428,8 @@ export interface GetQueryErrorResponseT {
 }
 
 export interface ErrorResponseT {
-  code: string; // To translate to localized messages
-  message?: string; // For developers / debugging only
-  context?: Record<string, string>; // More information to maybe display in translated messages
+  message?: string; // Localized error message (based on Accept-Language header) to show to users
```

Comment:
@awildturtok will `message` always be defined?

Reply:
@MaxVonKnobloch ?

Mine are all defined, so maybe we should enforce that also in ExternalForms